__tests__/utils/components/auto-complete-zipcode.test.js

```diff
@@ -83,4 +86,22 @@ describe("AutoCompleteZipcode Utils", () => {
       expect(result).toEqual(expected)
     })
   })
+
+  describe("isValidZipcode", () => {
+    test("Should return true when 44000", () => {
+      expect(isValidZipcode("44000")).toBeTruthy()
+    })
+    test("Should return false when 44 (too short)", () => {
+      expect(isValidZipcode("44")).toBeFalsy()
+    })
+    test("Should return false when 440000 (too long)", () => {
+      expect(isValidZipcode("440000")).toBeFalsy()
+    })
+    test("Should return false when aaaaa (not number)", () => {
+      expect(isValidZipcode("aaaaa")).toBeFalsy()
+    })
+    test("Should return trfalseue when aa (too short and not number)", () => {
```

Comment:
je pense qu'on peut écrire direct la RG du coup : 

`should return true when zip code has 5 digits`
`should return false when zip code has less than 5 digits`
`should return false when zip code has more than 5 digits`
`should return false when zip code is not a number and has 5 characters`
`should return false when zip code is not a number and has less than 5 characters`